### ISSUE TITLE
Remove the old GSD namespace and put the OSV data in there

### ISSUE THREAD

**Issue by joshbressers** [cloudsecurityalliance/gsd-database#2387]
In our old data, we have a GSD and OSV namespace. The GSD namespace is old and not a standard. The OSV namespace in most instances will be a mirror of the GSD data. We should get rid of the old GSD space.

I would prefer we do not use a namespace called OSV as I think it's not clear that it has nothing to do with osv.dev

**Comment by kurtseifried** [comment]
Agreed. my plan was to convert OSV->gsd int he files, and drop anything in the existing GSD namespace that isn't unique (e.g. if we have extra data there we'll deal with it, but if it's a carbon copy we just drop it).

**Comment by joshbressers** [comment]
We should drop it. Then we can rename the OSV namespace to GSD, as in this context OSV means format, not data owner, which is confusing
